src/commands/eval.js

```diff
@@ -1,28 +1,26 @@
-
-
 module.exports = {
   description: "Evaluate some code.",
-  options: [
-    {
-      type: 3,
-      name: "code",
-      description: "The code you want to run through the bot.",
-      required: true
-    }
-  ],
+  options: [{
+    type: 3,
+    name: "code",
+    description: "The code you want to run through the bot.",
+    required: true
+  }],
   aliases: [],
   permissionRequired: 7 // 0 All, 1 Assistant, 2 Helper, 3 Moderator, 4 Exec.Assistant, 5 Executive, 6 Director, 7 Promise#0001
 };
 
-module.exports.run = async ({ channel }, { code }) => {
+module.exports.run = async ({ channel, client }, { code }) => {
   try {
-    let evaled = eval(code);
+    let evaled = await eval(code);
     if (typeof evaled != "string") evaled = require("util").inspect(evaled);
+    evaled = evaled.replace(client.token, "-- Redacted ---");
+
```

Comment:
If I ever do leak the client token then I will simply refresh it. I actually fetch the client token through this command quite a lot in private channels, so I would like to keep this untouched.

Reply:
I added because it can be leaked without realising because its attached to the client class. I shall remove it tho